### ISSUE TITLE
Some mistake

### ISSUE THREAD

**Issue by CyberHplusplus** [yuyama137/influenza#1]
prepare_data.py  #utils.set_data() =>  utils.set_data()
modle.py y = torch.cat([y, y_pred[:,[-1],:]], dim=1) => y = torch.cat([y, y_pred[:,[-1],:]], dim=2)

**Comment by Ev1cH4n** [comment]
> I don't understand "prepare_data.py #utils.set_data() => utils.set_data()" part of your post.
Please teach me what wrong is.

**Comment by yuyama137** [comment]
Sorry for the delay.
Could you tell me more about both issues?

Well, I'm not familiar with English, so sorry if my meaning is not clear.

thank you.

**Comment by WUTmouse** [comment]
Hello，I want to ask how can I run your code in Windows？ Please teach me 。thx

**Comment by yuyama137** [comment]
Hello,
You can run this code by linux based system (environment which can use shell). In Windows, you may run by using WSL(Windows Subsystem for Linux).
I hope this helps to solve the problem.
thank you.

**Comment by WUTmouse** [comment]
> Hello, You can run this code by linux based system (environment which can use shell). In Windows, you may run by using WSL(Windows Subsystem for Linux). I hope this helps to solve the problem. thank you.

I tried it and now I've succeeded. Thank you for your help！

**Comment by soaperless** [comment]
Could I run this code in the Pycharm？

**Comment by yuyama137** [comment]
Hello,
I've never done it, but I think it's probably possible.

**Comment by soaperless** [comment]
> but
okay，thanks for answering
